client/sites-dashboard/components/site-status-badge.tsx

```diff
@@ -0,0 +1,32 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import SitesLaunchStatusBadge from './sites-launch-status-badge';
+
+interface SiteStatusBadgeProps {
+	site: SiteExcerptData;
+	as?: 'span' | typeof SitesLaunchStatusBadge;
+	hideLive?: boolean;
+}
+
+export const SiteStatusBadge = ( {
```

Comment:
The name of this component is misleading and confusing with `SitesLaunchStatusBadge`. Looking for suggestions.

Reply:
maybe SiteStatusIndicator